### PR TITLE
Rename Subspace to Autonomys

### PR DESCRIPTION
**Pull request by EmilFattakhov**
This PR renames the instanced of Subspace Network to Autononys Network while keeping:

- Referenced to Subspace Protocol where makes sense
- Referenced to subspace node and subspace farmer (they weren't renamed just yet)

It also updates the logos to the Autonomy logos.

### CODE REVIEW

**Comment by EmilFattakhov** [comment]
There's a high chance I missed something, but I think most of the instanced have been renamed.

File: versioned_docs/version-latest/develop/nova/general-information.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 
-For local development and testing, you have multiple options. You can spin up your own version of a Subspace Developer Node and farmer to deploy contracts, develop applications, and run tests. Alternatively, you can use Ethereum development tools like [Hardhat](https://hardhat.org/hardhat-network/docs/overview) or [Anvil](https://book.getfoundry.sh/anvil/), which are fully compatible with Subspace due to their EVM compatibility.
+For local development and testing, you have multiple options. You can spin up your own version of a Autonomys Developer Node and farmer to deploy contracts, develop applications, and run tests. Alternatively, you can use Ethereum development tools like [Hardhat](https://hardhat.org/hardhat-network/docs/overview) or [Anvil](https://book.getfoundry.sh/anvil/), which are fully compatible with Autonomys due to their EVM compatibility.
```
Comment (jim-counter) [inline versioned_docs/version-latest/develop/nova/general-information.md:18]:
```suggestion
For local development and testing, you have multiple options. You can spin up your own version of an Autonomys Developer Node and farmer to deploy contracts, develop applications, and run tests. Alternatively, you can use Ethereum development tools like [Hardhat](https://hardhat.org/hardhat-network/docs/overview) or [Anvil](https://book.getfoundry.sh/anvil/), which are fully compatible with Autonomys due to their EVM compatibility.
```

File: docs/develop/nova/general-information.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 
-For local development and testing, you have multiple options. You can spin up your own version of a Subspace Developer Node and farmer to deploy contracts, develop applications, and run tests. Alternatively, you can use Ethereum development tools like [Hardhat](https://hardhat.org/hardhat-network/docs/overview) or [Anvil](https://book.getfoundry.sh/anvil/), which are fully compatible with Subspace due to their EVM compatibility.
+For local development and testing, you have multiple options. You can spin up your own version of a Autonomys Developer Node and farmer to deploy contracts, develop applications, and run tests. Alternatively, you can use Ethereum development tools like [Hardhat](https://hardhat.org/hardhat-network/docs/overview) or [Anvil](https://book.getfoundry.sh/anvil/), which are fully compatible with Autonomys due to their EVM compatibility.
```
Comment (jim-counter) [inline docs/develop/nova/general-information.md:18]:
```suggestion
For local development and testing, you have multiple options. You can spin up your own version of an Autonomys Developer Node and farmer to deploy contracts, develop applications, and run tests. Alternatively, you can use Ethereum development tools like [Hardhat](https://hardhat.org/hardhat-network/docs/overview) or [Anvil](https://book.getfoundry.sh/anvil/), which are fully compatible with Autonomys due to their EVM compatibility.
```

File: docs/farming-&-staking/farming/additional-guides/grafana-dashboard.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Grafana Dashboard
 sidebar_position: 3
-description: How to configure the Grafana dashboard for a Subspace Network node
+description: How to configure the Grafana dashboard for a Autonomys Network node
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/additional-guides/grafana-dashboard.md:4]:
```suggestion
description: How to configure the Grafana dashboard for an Autonomys Network node
```

File: docs/farming-&-staking/farming/advanced-cli/cli-farming-cluster.mdx [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Farming Cluster
 sidebar_position: 4
-description: How to run a Subspace Network Farmer using the farming cluster
+description: How to run a Autonomys Network Farmer using the farming cluster
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/advanced-cli/cli-farming-cluster.mdx:4]:
```suggestion
description: How to run an Autonomys Network Farmer using the farming cluster
```

File: docs/farming-&-staking/farming/advanced-cli/cli-install.mdx [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Install
 sidebar_position: 2
-description: How to run a Subspace Network Farmer with the Substrate CLI
+description: How to run a Autonomys Network Farmer with the Substrate CLI
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/advanced-cli/cli-install.mdx:4]:
```suggestion
description: How to run an Autonomys Network Farmer with the Substrate CLI
```

File: docs/farming-&-staking/farming/advanced-cli/cli-troubleshooting.mdx [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Troubleshooting
 sidebar_position: 5
-description: How to run a Subspace Network Farmer with the Substrate CLI
+description: How to run a Autonomys Network Farmer with the Substrate CLI
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/advanced-cli/cli-troubleshooting.mdx:4]:
```suggestion
description: How to run an Autonomys Network Farmer with the Substrate CLI
```

File: docs/develop/nova/block_explorer.md [hunk 7]
```diff
@@ -1,17 +1,17 @@
 ---
-title: Subspace block explorer
+title: Autonomys block explorer
 sidebar_position: 10
-description: Subspace hosted block explorer
+description: Autonomys hosted block explorer
 keywords:
   - block explorer
-  - subspace network
+  - autononys network
```
Comment (jim-counter) [inline docs/develop/nova/block_explorer.md:7]:
```suggestion
  - autonomys network
```

File: docs/develop/nova/foundry_guide.md [hunk 8]
```diff
@@ -3,7 +3,7 @@ title: Foundry - testing and deployment
 sidebar_position: 7
 description: Testing and Deploying Smart Contracts using Foundry
 keywords:
-  - subspace network
+  - autononys network
```
Comment (jim-counter) [inline docs/develop/nova/foundry_guide.md:6]:
```suggestion
  - autonomys network
```

File: docs/develop/nova/general-information.md [hunk 9]
```diff
@@ -1,23 +1,23 @@
 ---
-title: General information on dev tools and Subspace EVM
+title: General information on dev tools and Autonomys EVM
 sidebar_position: 3
 description: Guide on setting up
 keywords:
   - metamask
-  - subspace network
+  - autononys network
```
Comment (jim-counter) [inline docs/develop/nova/general-information.md:7]:
```suggestion
  - autonomys network
```

File: docs/develop/nova/hardhat_guide.md [hunk 10]
```diff
@@ -3,13 +3,13 @@ title: Hardhat
 sidebar_position: 8
 description: Testing and Deploying Smart Contracts using HardHat
 keywords:
-  - subspace network
+  - autononys network
```
Comment (jim-counter) [inline docs/develop/nova/hardhat_guide.md:6]:
```suggestion
  - autonomys network
```

File: docs/develop/nova/quick_start.md [hunk 11]
```diff
@@ -3,20 +3,20 @@ title: Quick start
 sidebar_position: 2
 description: Quick start one-pager
 keywords:
-  - subspace network
+  - autononys network
```
Comment (jim-counter) [inline docs/develop/nova/quick_start.md:6]:
```suggestion
  - autonomys network
```

File: docs/develop/nova/remix_guide.md [hunk 12]
```diff
@@ -3,13 +3,13 @@ title: Remix IDE - testing and deployment
 sidebar_position: 6
 description: Testing and Deploying Smart Contracts using Remix IDE
 keywords:
-  - subspace network
+  - autononys network
```
Comment (jim-counter) [inline docs/develop/nova/remix_guide.md:6]:
```suggestion
  - autonomys network
```

File: docs/develop/nova/setting-up-metamask.md [hunk 13]
```diff
@@ -4,14 +4,14 @@ sidebar_position: 4
 description: Guide on setting up MetaMask for development purposes
 keywords:
   - metamask
-  - subspace network
+  - autononys network
```
Comment (jim-counter) [inline docs/develop/nova/setting-up-metamask.md:7]:
```suggestion
  - autonomys network
```

File: docs/farming-&-staking/farming/advanced-cli/cli-install.mdx [hunk 14]
```diff
@@ -34,7 +34,7 @@ There are multiple versions of our releases available below.
 ## Get Started with Advanced CLI Installation
 
 We will download two files for your respective operating system.
-1. `Subspace-Node` - This is the executable that actually makes the connection with the Subspace Network
+1. `Subspace-Node` - This is the executable that actually makes the connection with the Autononys network
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/advanced-cli/cli-install.mdx:37]:
```suggestion
1. `Subspace-Node` - This is the executable that actually makes the connection with the Autonomys network
```

File: versioned_docs/version-latest/develop/nova/block_explorer.md [hunk 15]
```diff
@@ -1,17 +1,17 @@
 ---
-title: Subspace block explorer
+title: Autonomys block explorer
 sidebar_position: 10
-description: Subspace hosted block explorer
+description: Autonomys hosted block explorer
 keywords:
   - block explorer
-  - subspace network
+  - autononys network
```
Comment (jim-counter) [inline versioned_docs/version-latest/develop/nova/block_explorer.md:7]:
```suggestion
  - autonomys network
```

File: versioned_docs/version-latest/develop/nova/foundry_guide.md [hunk 16]
```diff
@@ -3,7 +3,7 @@ title: Foundry - testing and deployment
 sidebar_position: 7
 description: Testing and Deploying Smart Contracts using Foundry
 keywords:
-  - subspace network
+  - autononys network
```
Comment (jim-counter) [inline versioned_docs/version-latest/develop/nova/foundry_guide.md:6]:
```suggestion
  - autonomys network
```

File: versioned_docs/version-latest/develop/nova/general-information.md [hunk 17]
```diff
@@ -1,23 +1,23 @@
 ---
-title: General information on dev tools and Subspace EVM
+title: General information on dev tools and Autonomys EVM
 sidebar_position: 3
 description: Guide on setting up
 keywords:
   - metamask
-  - subspace network
+  - autononys network
```
Comment (jim-counter) [inline versioned_docs/version-latest/develop/nova/general-information.md:7]:
```suggestion
  - autonomys network
```

File: versioned_docs/version-latest/develop/nova/hardhat_guide.md [hunk 18]
```diff
@@ -3,13 +3,13 @@ title: Hardhat
 sidebar_position: 8
 description: Testing and Deploying Smart Contracts using HardHat
 keywords:
-  - subspace network
+  - autononys network
```
Comment (jim-counter) [inline versioned_docs/version-latest/develop/nova/hardhat_guide.md:6]:
```suggestion
  - autonomys network
```

File: versioned_docs/version-latest/develop/nova/quick_start.md [hunk 19]
```diff
@@ -3,20 +3,20 @@ title: Quick start
 sidebar_position: 2
 description: Quick start one-pager
 keywords:
-  - subspace network
+  - autononys network
```
Comment (jim-counter) [inline versioned_docs/version-latest/develop/nova/quick_start.md:6]:
```suggestion
  - autonomys network
```

File: versioned_docs/version-latest/develop/nova/remix_guide.md [hunk 20]
```diff
@@ -3,13 +3,13 @@ title: Remix IDE - testing and deployment
 sidebar_position: 6
 description: Testing and Deploying Smart Contracts using Remix IDE
 keywords:
-  - subspace network
+  - autononys network
```
Comment (jim-counter) [inline versioned_docs/version-latest/develop/nova/remix_guide.md:6]:
```suggestion
  - autonomys network
```

File: versioned_docs/version-latest/develop/nova/setting-up-metamask.md [hunk 21]
```diff
@@ -4,14 +4,14 @@ sidebar_position: 4
 description: Guide on setting up MetaMask for development purposes
 keywords:
   - metamask
-  - subspace network
+  - autononys network
```
Comment (jim-counter) [inline versioned_docs/version-latest/develop/nova/setting-up-metamask.md:7]:
```suggestion
  - autonomys network
```

**Review by jim-counter — CHANGES_REQUESTED** [review]
Mainly small typos.

**Comment by EmilFattakhov** [comment]
I'm around for a bit @jim-counter if you need help to polish the PR, but looks like you're mostly done here :) Thanks for the review!

File: docs/farming-&-staking/wallets/polkadot.md [hunk 22]
```diff
@@ -81,19 +81,19 @@ If you face any trouble or would like to learn about other features for Polkadot
 
     ![trouble-1](/img/doc-imgs/polkadot/trouble-1.png)
 
-- You can see your Subspace Testnet public address via the `...` menu and setting the `Allow Use on Any Chain` dropdown to `Subspace Testnet`, once you exit you will see the public address now starts with `st`
+- You can see your Autonomys Testnet public address via the `...` menu and setting the `Allow Use on Any Chain` dropdown to `Autonomys Testnet`, once you exit you will see the public address now starts with `st`
```
[the remark this answers is not in the document]
Comment (EmilFattakhov) [reply docs/farming-&-staking/wallets/polkadot.md]: Fixed 👍

File: docs/farming-&-staking/wallets/subwallet.md [hunk 23]
```diff
@@ -1,7 +1,7 @@
 ---
 title: SubWallet (Recommended)
 sidebar_position: 1
-description: How to configure the SubWallet Substrate wallet for the Subspace Network
+description: How to configure the SubWallet Substrate wallet for the Autonomys Network
```
[the remark this answers is not in the document]
Comment (EmilFattakhov) [reply docs/farming-&-staking/wallets/subwallet.md]: Also addressed 👍

**Comment by EmilFattakhov** [comment]
All comments were addressed @jim-counter, please have another look!

**Pull request merged**